roofit/roofitcore/test/RooGradMinimizer.cxx

```diff
@@ -0,0 +1,847 @@
+/*****************************************************************************
```

Comment:
The convention of the test name files is to call it `test<Class>.cxx, so here the name of the file should be `testRooGradMinimizerFcn.cxx` I'd say.